lambdas/scripts/bulk_upload_column_change.py

```diff
@@ -7,32 +7,31 @@
 from pydantic import BaseModel, TypeAdapter
 from services.base.dynamo_service import DynamoDBService
 
-COLUMNS_TO_FETCH = "OdsCode,PdsOdsCode,UploaderOdsCode,ID"
-ATTRIBUTE_TO_REMOVE = "OdsCode"
-ATTRIBUTE_TO_ADD = "PdsOdsCode"
-UPLOADER_ODS_CODE = "PlaceHolder"
+TABLE = "ndrb_BulkUploadReport"
```

Comment:
This is targeting a specific sandbox's table. Shouldn't this be modified dynamically depending on the environment? i.e `{env}_BulkUploadReport`

Reply:
I spoke with Noga and she's happy to keep it as-is as the script will be run infrequently. As a side note, it seems like {env} isn't an environment variable I could find in the codebase, it wouldn't be a trivial change from what I could see.